modules/apps/social/social-activity-test-util/src/main/java/com/liferay/social/activity/test/util/BaseSocialActivityInterpreterTestCase.java

```diff
@@ -176,14 +158,37 @@ protected void checkLinks() throws Exception {
 		}
 	}
 
+	protected void checkRenaming(List<SocialActivity> originalActivities)
+		throws Exception {
+
+		Assert.assertFalse(
+			originalActivities.toString(), originalActivities.isEmpty());
+
+		String originalTitle = _getActivitiesFirstTitle(originalActivities);
+		Set<Long> originalIds = _getActivitiesIds(originalActivities);
+
+		List<SocialActivity> activities = getActivities();
+
+		Assert.assertFalse(activities.toString(), activities.isEmpty());
+
+		for (SocialActivity activity : activities) {
+			if (!originalIds.contains(activity.getActivityId())) {
+				String title = activity.getExtraDataValue(
+					"title", serviceContext.getLocale());
+
+				if (isSupportsRename(activity.getClassName()) &&
+					Validator.isNotNull(title)) {
+
+					Assert.assertNotEquals(originalTitle, title);
+				}
+			}
+		}
+	}
```

Comment:
@marcogalluzzi Does this need to be protected? If subclasses are not expected to override it, it would be better to make it private.

Reply:
I've just used the visibility defined in other `check*` methods. Thinking about it, it doesn't make too much sense for these methods to be protected. We could change their visibility to `private`, I've just seen that there are no overriding implementations. Should I do this change?

Reply:
Yes, go ahead. Also, remember to run `gw baseline` in the module so that the bnd and packageinfo files are updated.

Reply:
OK done